.travis.yml

```diff
@@ -1,20 +1,24 @@
+os:
+  - linux
+  - osx
+
 language: c++
 
 compiler:
     - gcc
 
 before_install:
-  - sudo apt-get update
-  - sudo apt-get install -y git
-  - sudo apt-get install -y libboost-python-dev
-  - sudo apt-get install -y libboost-system-dev
-
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then sudo apt-get update; fi
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then sudo apt-get install -y git; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install boost-python; else sudo apt-get install -y libboost-python-dev; fi
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then sudo apt-get install -y libboost-system-dev; fi
 before_script:
   - mkdir build
   - cd build
-  - cmake ../src
-
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cmake ../src -DPYTHON_LIBRARY=$(python-config --prefix)/lib/libpython2.7.dylib -DPYTHON_INCLUDE_DIR=$(python-config --prefix)/include/python2.7 -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev; else cmake ../src; fi
 script: 
   - make
 
 after_success:
+  - sudo make install
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then otool -L /usr/local/lib/python2.7/site-packages/ocl.so; fi
```

Comment:
I added this command to inspect if ocl is linked to the right libs in the travis build, we should probably add the same for linux and run some python scripts to see if things are working correctly